### DISCUSSION TITLE
Vault sync with collections

### DISCUSSION THREAD

**Discussion by mvalois** [dani-garcia/vaultwarden#2203] [Q&A] [answered]
Hello,
It looks like vault auto-sync, using websocket, works only for added items and not for added collections. I've tried adding a collection from the web vault and it does not appear automatically in the firefox web extension (I have to manually resync).
Is it an expected behavior?

**Comment by BlackDex** [comment, marked as answer]
Unfortunately it is.
This is the same for Upstream Bitwarden.
websocket sync for collections and org items isn't working.
Though i haven't checked it out for a while now on the latest version of Bitwarden it self

**Comment by mvalois** [comment]
Thank you for your quick answer!
Are you aware of any technical difficulty to implement it?

**Reply by BlackDex** [reply]
It's a limitation of the web-vault. It just doesn't respond to the websocket sync requests what i can remember.
So, its not a server side item, but more a client-side item.

If i'm correct it had something to do with not being able to filter that well between all the users who does and does not have access to a specific collection. But again, i have not looked at it for a while.
